Tests/MeiliSearchIntegrationTests/KeysTests.swift

```diff
@@ -0,0 +1,177 @@
+  }
+
+  func testCreateKey() {
+    let keyException = XCTestExpectation(description: "Create a key")
+
+    let keyParams = KeyParams(description: "Custom", actions: ["*"], indexes: ["*"], expiresAt: nil)
+    self.client.createKey(keyParams) { result in
+      switch result {
+      case .success(let key):
+        XCTAssertEqual(key.expiresAt, nil)
+        keyException.fulfill()
+      case .failure(let error):
+        dump(error)
+        XCTFail("Failed to create a key")
+        keyException.fulfill()
+      }
+    }
+
+    self.wait(for: [keyException], timeout: 10.0)
+  }
+
+  func testCreateKeyWithExpire() {
+    let keyException = XCTestExpectation(description: "Create a key with an expireAt value")
+
+    let formatter = DateFormatter()
+    formatter.dateFormat = "yyyy-MM-dd"
+    let someDateTime = formatter.string(from: Date.distantFuture)
+
+    let keyParams = KeyParams(description: "Custom", actions: ["*"], indexes: ["*"], expiresAt: someDateTime)
+    self.client.createKey(keyParams) { result in
+      switch result {
+      case .success(let key):
+        XCTAssertNotNil(key.expiresAt)
+        keyException.fulfill()
+      case .failure(let error):
+        dump(error)
+        XCTFail("Failed to create a key")
+        keyException.fulfill()
+      }
+    }
+    self.wait(for: [keyException], timeout: 10.0)
+  }
+
+  func testUpdateKey() {
+    let keyException = XCTestExpectation(description: "Update a key")
+
+    let keyParams = KeyParams(description: "Custom", actions: ["*"], indexes: ["*"], expiresAt: nil)
+    self.client.createKey(keyParams) { result in
+      switch result {
+      case .success(let key):
+        let formatter = DateFormatter()
+        formatter.dateFormat = "yyyy-MM-dd"
+        let newDate = formatter.string(from: Date.distantFuture)
+        let keyParams = KeyParams(description: "Custom", actions: ["*"], indexes: ["*"], expiresAt: newDate)
+        self.client.updateKey(key: key.key, keyParams: keyParams) { result in
+          switch result {
+          case .success(let key):
+            XCTAssertNotNil(key.expiresAt)
+            keyException.fulfill()
+          case .failure(let error):
+            dump(error)
+            XCTFail("Failed to update a key")
+            keyException.fulfill()
+          }
+        }
+      case .failure(let error):
+        dump(error)
+        XCTFail("Failed to create a key")
+        keyException.fulfill()
+      }
+    }
+    self.wait(for: [keyException], timeout: 10.0)
+  }
+
+  func testDeleteKey() {
+    let keyException = XCTestExpectation(description: "Delete a key")
+
+    let keyParams = KeyParams(description: "Custom", actions: ["*"], indexes: ["*"], expiresAt: nil)
+    self.client.createKey(keyParams) { result in
+      switch result {
+      case .success(let key):
+        self.client.deleteKey(key: key.key) { result in
+          switch result {
+          case .success:
+            keyException.fulfill()
```

Comment:
In this test shouldn't we test a `self.client.getKey` in order to check if the key was really removed?